src/cli.rs

```diff
@@ -13,19 +13,34 @@ pub struct Args {
     pub target: Option<Target>,
     pub target_dir: Option<PathBuf>,
     pub docker_in_docker: bool,
+    pub project_dir: Option<PathBuf>,
 }
 
 pub fn parse(target_list: &TargetList) -> Args {
     let mut channel = None;
     let mut target = None;
+    let mut project_dir: Option<PathBuf> = None;
```

Comment:
Can you make this be the `manifest_path` instead? That way we could use the same logic whether `--manifest-path` is specified or not. I. e. automatically find the `Cargo.toml` when running `cross` in a subdirectory as well.

Reply:
Not sure I understand. Do you mean just renaming the variable?
`project_dir` is not necessarily `manifest_path`. That's the point of this PR to decouple the mounted root path from the location of the `Cargo.toml` file. So I'd say it's better to have a distinctive name for it.

Reply:
> `project_dir` is not necessarily `manifest_path`

From looking at the code, `project_dir` is `current_dir`/`manifest_path`, i.e. the absolute path to the `Cargo.toml`, isn't it?

Reply:
Right, I was confused.